### PR TITLE
Feature/ergo cron

### LINKED ISSUE #34: ergonode_php_1: setfacl: var: Not supported
It looks like the main php container (`ergonode_php_1`) exits on start. Here are the logs:

```console
$ docker logs ergonode_php_1
Linking /usr/local/etc/php/php.ini-development > /usr/local/etc/php/php.ini
Verified OK
Setting file permissions...
setfacl: var: Not supported
setfacl: var/cache: Not supported
setfacl: var/log: Not supported
```

..after that, it exits. Here's a full `ps -a` that shows what works and what doesn't:
```console
$ docker ps -a
CONTAINER ID        IMAGE                      COMMAND                  CREATED             STATUS                        PORTS               NAMES
ba4b5c749a8f        ergonode/nginx:latest      "docker-entrypoint n…"   4 minutes ago       Exited (143) 12 seconds ago                       ergonode_nginx_1
e52b15cde028        ergonode/php:latest        "docker-entrypoint b…"   4 minutes ago       Exited (6) 4 minutes ago                          ergonode_php-messenger-channel_1
78420e045f80        ergonode/php:latest        "docker-entrypoint b…"   4 minutes ago       Exited (6) 4 minutes ago                          ergonode_php-messenger-import_1
8384deebe73d        ergonode/php:latest        "docker-entrypoint b…"   4 minutes ago       Exited (6) 4 minutes ago                          ergonode_php-messenger-event_1
4352a72c70cb        ergonode/php:latest        "docker-entrypoint p…"   4 minutes ago       Exited (1) 4 minutes ago                          ergonode_php_1
57fdb598ef1b        ergonode/php:latest        "docker-entrypoint b…"   4 minutes ago       Exited (6) 4 minutes ago                          ergonode_php-messenger-export_1
4575b2b6636c        ergonode/php:latest        "docker-entrypoint b…"   4 minutes ago       Exited (6) 4 minutes ago                          ergonode_php-messenger-segment_1
a2563fcf1097        ergonode/php:latest        "docker-entrypoint b…"   4 minutes ago       Exited (6) 4 minutes ago                          ergonode_php-messenger-core_1
479506a29cfb        ergonode/rabbitmq:latest   "docker-ergonode-ent…"   4 minutes ago       Exited (137) 2 seconds ago                        ergonode_rabbitmq-02_1
707f0cfdd73d        ergonode/rabbitmq:latest   "docker-ergonode-ent…"   4 minutes ago       Exited (137) 2 seconds ago                        ergonode_rabbitmq-03_1
f58536f6f5a8        ergonode/rabbitmq:latest   "docker-ergonode-ent…"   4 minutes ago       Exited (137) 2 seconds ago                        ergonode_rabbitmq-01_1
f552956a83e2        ergonode/postgres:latest   "docker-entrypoint.s…"   4 minutes ago       Exited (0) 12 seconds ago                         ergonode_postgres_1
584d6fb9f42c        ergonode/nuxtjs:latest     "docker-entrypoint n…"   4 minutes ago       Exited (137) 2 seconds ago                        ergonode_nuxtjs_1
902861d384e2        ergonode/haproxy:latest    "/docker-entrypoint.…"   4 minutes ago       Exited (1) 4 minutes ago                          ergonode_haproxy_1
a46cfb79026f        consul:1.7                 "docker-entrypoint.s…"   4 minutes ago       Exited (1) 1 second ago                           ergonode_consul_1
d246fa2e582b        ergonode/docs:latest       "docker-entrypoint d…"   4 minutes ago       Exited (137) 2 seconds ago                        ergonode_docsify_1
```

### PR DESCRIPTION
**Pull request by Daniel-Marynicz**

### CODE REVIEW

File: config/php/docker-entrypoint.sh [hunk 1]
```diff
@@ -162,6 +162,45 @@ if [ "$1" = 'php-fpm' ] ; then
     >&2 echo "app initialization finished"
 fi
 
+if [ "$1" = 'crond' ] ; then
+    mkdir -p var/cache var/log public/multimedia public/thumbnail public/avatar import export
+    >&2 echo "Setting file permissions..."
+    setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX var
```
Comment (piotrkreft) [inline config/php/docker-entrypoint.sh]: Could we start avoiding ACL's usage as it's not supported on osxfs?

**Review by piotrkreft — CHANGES_REQUESTED** [review]

File: config/php/docker-entrypoint.sh (hunk 1 above)
Comment (Daniel-Marynicz) [reply config/php/docker-entrypoint.sh]: ok i can add some if for this

File: config/php/docker-entrypoint.sh [hunk 2]
```diff
@@ -62,6 +62,32 @@ function waitUntil() {
   done
 }
 
+function setPermissions()
```
Comment (piotrkreft) [inline config/php/docker-entrypoint.sh]: I'd suggest function name be more like `createApplicationDirs` or `createVarDirs` - setting permissions is just a way to achieve the effect, not the intention itself

File: config/php/docker-entrypoint.sh [hunk 3]
```diff
@@ -62,6 +62,32 @@ function waitUntil() {
...
+{
+  mkdir -p var/cache var/log public/multimedia public/thumbnail public/avatar import export
+  >&2 echo "Setting file permissions..."
+  if setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX var ; then
```
Comment (piotrkreft) [inline config/php/docker-entrypoint.sh]: Could we not go recursive on condition?
It'd generate **a lot** of error logs about every single file in dir not being supported and therefore could lead to confusion whether everything is correct.
Alternatively, we could verify fs itself with sth like `df`.
Up to you

**Review by piotrkreft — CHANGES_REQUESTED** [review]

File: config/php/docker-entrypoint.sh [hunk 4]
```diff
@@ -124,22 +150,7 @@ if [[ "$1" =~ bin/console ]] && [[ "$2" = 'messenger:consume' ]]; then
...
-    setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX public/thumbnail
-    setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX public/avatar
-    setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX public/avatar
-    setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX import
-    setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX import
-    setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX export
-    setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX export
-
-
+    setPermissions
```
Comment (piotrkreft) [inline config/php/docker-entrypoint.sh]: 👍  for tackling the existing issue

**Review by piotrkreft — APPROVED** [review]

**Pull request merged**